### ISSUE TITLE
setLocale method does not work (android)

### ISSUE THREAD

**Issue by Edmu** [Instabug/instabug-cordova#86]
Platform android:

```
cordova.plugins.instabug.setLocale(
  'german', 
  ok => console.log('succ', ok), 
  err => console.log('err', err)
);
```

Output:

`err Invalid action`

**Comment by AliAbdelfattah** [comment]
Thanks for reporting this, @Edmu. Which platform are you testing on?

**Comment by Edmu** [comment]
android (stated in the issue title)

**Comment by AliAbdelfattah** [comment]
Oh, sorry for missing that. We'll look into it.

**Comment by alyezz** [comment]
@Edmu this issue has been fixed and is included in out latest release 9.1.6
